stl/src/charconv.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This must be as small as possible, because its contents are
+// injected into the msvcprt.lib and msvcprtd.lib import libraries.
+// Do not include or define anything else here.
+// In particular, basic_string must not be included here.
+
+#include <charconv>

```

Comment:
It would be nice to avoid dragging `<charconv>` into the import lib - although I'm not aware of any IDL sensitivity (which would be doom, as the comment above notes), it is very large and may grow over time. This is being done for `_General_precision_tables` - could we convert those into non-member arrays (like `_Large_power_data`) and slightly change `<charconv>` accordingly, so we only need `<cstdint>` here? For example:

* Rename `_General_precision_tables` to `_General_precision_tables2` (to avoid any ABI concerns), and
* Change its arrays to be either references bound to non-member arrays (if possible), or pointers (less preferable as that loses bounds info, not that we're worrying about out-of-bounds access), or functions returning references to the non-member arrays?